### ISSUE TITLE
__.List() Question

### ISSUE THREAD

**Issue by g3rd** [emehrkay/Pypher#10]
Why does the `List` join with a `'  '`  and not a `', '`?

See: https://github.com/emehrkay/Pypher/blob/master/pypher/builder.py#L670

When I do:

```
...
p.Return.apoc_text_join(__.List(__.u.__first_name__, __.u.__last_name__), "' '"))
...
```

I'm getting:

```
RETURN apoc.text.join([u.`first_name` u.`last_name`), " ")
```

**Comment by emehrkay** [comment]
I think you’re right. It should join on a comma. Would you mind fixing it (and the tests) and I’ll pull it in a little bit later?

**Comment by emehrkay** [comment]
This should be fixed in 0.9.2

**Comment by g3rd** [comment]
You beat me to it!
